### DISCUSSION TITLE
func.yaml documentation

### DISCUSSION THREAD

**Discussion by darthkali** [knative/func#1766] [Q&A] [answered]
Hello everyone,

I'm currently working with Knative functions, specifically using the func.yaml configuration file. Unfortunately, I've been unable to locate any detailed documentation on it. I'm keen to understand more about it.

Does anyone know where I can find a comprehensive guide or resources about the func.yaml in Knative? Any help is greatly appreciated.

Thank you!

**Comment by zroubalik** [comment, marked as answer]
Hi, is this helpful? https://github.com/knative/func/blob/main/docs/reference/func_yaml.md

**Reply by darthkali** [reply]
Thanks for the Link. I already found this. Its a good starting point but I think there is a lot of Improvement nessesary

**Comment by lkingland** [comment]
We're starting a full documentation update now, and with regards to this specific issue, the `func.yaml` structure and fields, a [dedicated discussion](https://github.com/knative/func/discussions/1878) is now open.
